network-traversal-quickstart/javascript/sample.env

```diff
@@ -0,0 +1,3 @@
+# Used in most samples. Retrieve these values from a Communication Services instance
```

Comment:
is there a public doc that can be linked here that shows where to find this in the portal? for these types of samples it's usually best to write them from the perspective of someone brand new to all of this so that no one is left out/confused

Reply:
can this comment be expanded to describe why/what this is used for?